bee-node/bee-node/src/tools/convert.rs

```diff
@@ -0,0 +1,105 @@
+// Copyright 2020-2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use bee_block::address::{Address, Ed25519Address};
+use crypto::hashes::{blake2b::Blake2b256, Digest};
+use structopt::StructOpt;
+use thiserror::Error;
+
+const BECH32_HRP: &str = "iota";
```

Comment:
This needs to be provided through the CLI, this is even wrong atm since this is the shimmer branch.